DependencyInjection/JMSSerializerExtension.php

```diff
@@ -30,6 +30,10 @@ class JMSSerializerExtension extends ConfigurableExtension
 {
     public function loadInternal(array $config, ContainerBuilder $container)
     {
+        $container
```

Comment:
`if (method_exists($container, 'registerForAutoconfiguration')) {
            $container
                ->registerForAutoconfiguration(EventSubscriberInterface::class)
                ->addTag('jms_serializer.event_subscriber');
        }`

Should fix the issue with sf  < 3.3

Reply:
Thx @juntereiner for your idea